client/go/cmd/testdata/tests/expected-suite.out

```diff
@@ -1,8 +1,11 @@
-Running my test: .... OK
-Running wrong-bool-value.json: Failed step 1:
-Unexpected value at /root/coverage/full: true
+My test: .... OK
+wrong-bool-value.json: failed
+Step 1:
+Unexpected value at /root/coverage/full
```

Comment:
Unexpected value **in response** at ...etc ?

Reply:
Reponse has unexpected value at ... ?

Reply:
Otherwise the "unexpeccted value" becomes too detached from the relevant path.

Reply:
But that's just wordy, again ...

Reply:
LGTM

Reply:
Or perhaps `In step 1` without name, and `In step 1: <name>` with name?  That way people also learn what the step number means.

Reply:
That's a great idea!

Reply:
Does it make sense to do this with files, too?

Reply:
How would that look?

Reply:
```feed-and-search-test.json: Test feeding and searching basics: ... OK```

Reply:
Perhaps too much.

Reply:
I suggest the stem of the filename **is** the test name, which avoids the duality and possibly duplicated prefix (<filename>:<testname>).  But we can revisit this later.